api/apiFunctions.test.tsx

```diff
@@ -21,297 +19,375 @@ jest.mock('./apiFunctions', () => {
     getAllLeagues: jest.fn(),
     getUserDocumentId: jest.fn(),
     addUserToLeague: jest.fn(),
+    getCurrentLeague: jest.fn(),
+    loginAccount: jest.fn(),
   };
 });
 
-describe('Auth Functions', () => {
-  jest.mock('./apiFunctions', () => ({
-    loginAccount: jest.fn(),
-    registerAccount: jest.fn(),
-  }));
-
-  xdescribe('login account successful', () => {
-    it('should show user login successfully', async () => {
-      const userDummy = {
-        email: 'testemail@email.com',
-        password: 'test1234',
-      };
-      await loginAccount(userDummy);
-      expect(account.createEmailPasswordSession).toBeInstanceOf(Object);
```

Comment:
Should we be removing these tests? Instead, could we move them elsewhere?

Reply:
why would we move them elsewhere if it isn't being used?